docs/sanic/class_based_views.md

```diff
@@ -92,10 +92,27 @@ class ViewWithDecorator(HTTPMethodView):
   def get(self, request, name):
     return text('Hello I have a decorator')
 
+  def post(self, request, name):
+    return text("Hello I also have a decorator")
+
 app.add_route(ViewWithDecorator.as_view(), '/url')
 ```
 
-#### URL Building
+But if you just want to decorator some functions and not all functions, you can use as followed.
```

Comment:
Can you please make a small change to make this say `But if you just want to decorate some functions and not all functions, you can do as follows:` instead?

Reply:
Of course.